tests/models/test_getmodel_unfusestrategy.py

```diff
@@ -0,0 +1,139 @@
+import pytest
+import torch
+import tempfile
+from fms.models import get_model
+
+# How to set unfuse_strategy to obtain desired `target_model` fusion
+#
+# ckpt       target_model   unfuse_strategy   tested
+#                            FP16    GPTQ
+# --------------------------------------------------
+# none       fused           None    None     (Y, Y)
+# none       unfused         post    pre      (Y, Y)
+# fused      fused           None    None     (Y, N)
+# fused      unfused         post    n/a      (Y, N)
+# unfused    fused           None    n/a      (Y, N)
+# unfused    unfused         post    pre      (Y, N)
+
+
+# FP16 model
+unfuse_strategies = [None, "post"]
+unfuse_ids = ["unfuse=None", "unfuse=post"]
+# gptq model
+unfuse_strategies_gptq = [None, "pre"]
+unfuse_ids_gptq = ["unfuse=None", "unfuse=pre"]
+
+expected_layers_from_fusion = {
+    "fused": [".qkv_fused.", ".wg1_fused."],
+    "unfused": [".query.", ".key.", ".value.", ".w1.", ".wg."],
+}
+
+
+class TestUnfuseStrategy:
```

Comment:
I don't think we need a class for this

Reply:
I organized these tests under classes because while for now they live in their own file, @ani300 initial recommendation was to add these tests to `test_models.py`. So, I wanted to keep fixture scope separate from the rest, in case these new gptq tests are eventually added to the set of existing ones.